src/evict/evict_lru.c

```diff
@@ -1575,6 +1586,26 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, u_int pct_full)
 	/* NOTREACHED */
 }
 
+/*
+ * __wt_evict_priority_set --
+ *	Set a tree's eviction priority.
+ */
+void
+__wt_evict_priority_set(WT_SESSION_IMPL *session, uint64_t v)
+{
+	S2BT(session)->evict_priority = v;
+}
+
+/*
+ * __wt_evict_priority_clear --
+ *	Clear a tree's eviction priority.
+ */
+void
+__wt_evict_priority_clear(WT_SESSION_IMPL *session)
+{
```

Comment:
What does clearing the priority mean? Presumably if we have a priority system, every handle should start with a given priority. I think some defined values here would make the priority more obvious. Something like:

```
#define WT_BTREE_EVICT_PRIORITY_DEFAULT 5
#define WT_BTREE_EVICT_PRIORITY_MAX 10
#define WT_BTREE_EVICT_PRIORITY_MIN 0
```

The any call you have to __wt_evict_priority_clear would become ``__wt_evict_priority_set(WT_BTREE_EVICT_PRIORITY_DEFAULT)``.